<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDrawLine</class>
 <widget class="QMainWindow" name="QDrawLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desde Vertice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dikstra">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar Algoritmo de Dijkstra</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertice1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color :rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasta Vertice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botoncrear">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Grafo Random </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>El camino más corto es de:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertice2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color :rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>810</width>
      <height>109</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Dirección IP del Servidor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineIP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Puerto del Servidor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linePUERTO"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="conectar">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin Conección</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
